tests/network/localnet/conftest.py

```diff
@@ -57,62 +65,196 @@ def vlan_id(vlan_index_number: Generator[int]) -> int:
+        interfaces=[
+            libnncp.Interface(
+                name=bridge_name,
+                type=NNCP_INTERFACE_TYPE_OVS_BRIDGE,
+                ipv4=libnncp.IPv4(enabled=False),
+                ipv6=libnncp.IPv6(enabled=False),
+                state=libnncp.Resource.Interface.State.UP,
+                bridge=libnncp.Bridge(
+                    options=libnncp.BridgeOptions(libnncp.STP(enabled=False)),
+                    port=[
+                        libnncp.Port(
+                            name=nodes_available_nics[worker_node1.name][-1],
+                        )
+                    ],
+                ),
+            )
+        ],
+        ovn=libnncp.OVN([
+            libnncp.BridgeMappings(
+                localnet=LOCALNET_OVS_BRIDGE_NETWORK,
+                bridge=bridge_name,
+                state=libnncp.BridgeMappings.State.PRESENT.value,
+            )
+        ]),
+    )
+    with libnncp.NodeNetworkConfigurationPolicy(
+        name=bridge_name,
+        desired_state=desired_state,
+        node_selector={WORKER_NODE_LABEL_KEY: ""},
+    ) as nncp:
+        nncp.wait_for_status_success()
+        yield nncp
+
+
+@pytest.fixture(scope="module")
+def nad_localnet_ovs_bridge(
+    namespace_localnet_1: Namespace,
+    vlan_id: int,
+) -> Generator[netattachdef.NetworkAttachmentDefinition]:
+    with localnet_nad(
+        namespace=namespace_localnet_1.name,
+        name=f"{LOCALNET_OVS_BRIDGE_NETWORK}-nad",
+        vlan_id=vlan_id,
+        network_name=LOCALNET_OVS_BRIDGE_NETWORK,
+    ) as nad:
+        yield nad
+
+
+@pytest.fixture(scope="module")
+def vm1_ovs_bridge_localnet(
+    ipv4_localnet_address_pool: Generator[str],
+    nad_localnet_ovs_bridge: netattachdef.NetworkAttachmentDefinition,
+) -> Generator[BaseVirtualMachine]:
+    with localnet_vm(
+        namespace=nad_localnet_ovs_bridge.namespace,
+        name="localnet-ovs-vm1",
+        nad_name=nad_localnet_ovs_bridge.name,
+        cidr=next(ipv4_localnet_address_pool),
+        spec_logical_network=LOCALNET_OVS_BRIDGE_NETWORK,
+    ) as vm:
+        yield vm
+
+
+@pytest.fixture(scope="module")
+def vm2_ovs_bridge_localnet(
+    ipv4_localnet_address_pool: Generator[str],
+    nad_localnet_ovs_bridge: netattachdef.NetworkAttachmentDefinition,
+) -> Generator[BaseVirtualMachine]:
+    with localnet_vm(
+        namespace=nad_localnet_ovs_bridge.namespace,
+        name="localnet-ovs-vm2",
+        nad_name=nad_localnet_ovs_bridge.name,
+        cidr=next(ipv4_localnet_address_pool),
+        spec_logical_network=LOCALNET_OVS_BRIDGE_NETWORK,
+    ) as vm:
+        yield vm
+
+
+@pytest.fixture(scope="module")
+def ovs_bridge_server_client_localnet_vms(
+    vm1_ovs_bridge_localnet: BaseVirtualMachine, vm2_ovs_bridge_localnet: BaseVirtualMachine
+) -> Generator[tuple[BaseVirtualMachine, BaseVirtualMachine]]:
+    server_vm, client_vm = run_vms(vms=[vm1_ovs_bridge_localnet, vm2_ovs_bridge_localnet])
```

Comment:
ditto

Reply:
Let's discuss this in the first comment, and I'll change it here too if needed.